src/styles/getMuiTheme.js

```diff
@@ -4,6 +4,21 @@ import ColorManipulator from '../utils/color-manipulator';
 import autoPrefix from './auto-prefix';
 import lightBaseTheme from './baseThemes/lightBaseTheme';
 import zIndex from './zIndex';
+import {autoprefixer, rtl} from './transformers';
+import compose from 'lodash.flowright';
+import warning from 'warning';
+
+const CALLED_ONCE = Symbol('prepareStyles/calledOnce');
```

Comment:
Mm, interesting. What do you and @alitaheri  prefer? Use a regular string key or polyfill?

I wonder we're inevitably going to keep running into `Symbol` issues because developers often assume it's "included".